packages/backend/src/apps/lettersg/actions/create-letter/index.ts

```diff
@@ -101,15 +145,17 @@ const action: IRawAction = {
           $.app.name,
         )
       }
-      // if not all fields are used
-      const lettersErrorData: LettersApiErrorData = error.response.data
-      if (lettersErrorData.message === 'Malformed bulk create object') {
-        throw new StepError(
-          'Missing pair of field/value for letter template',
-          'Click on set up action and check that you have entered all the fields and values in the letter parameters.',
-          $.step.position,
-          $.app.name,
-        )
+      // TODO (mal): check specific fields to return
+      if (error.response.status === 400) {
```

Comment:
nit: Since this is only for the "create letter" endpoint and not the PDF endpoint, can we shift this `catch` up to before we call the pdf endpoint.

Concern is that we don't want to parse error wrongly if the PDF endpoint return HTTP 400 in the future.

Reply:
so two separate try catches? but then I have no specific errors I know to catch for PDF endpoint tho

Reply:
lemme check for http error instance here instead actly